ert_storage/database_schema.py

```diff
@@ -81,6 +81,7 @@ def __init__(
     )
 
     name = sa.Column(sa.String, nullable=False)
+    _metadata = sa.Column(sa.JSON, nullable=True)
```

Comment:
Perhaps it's possible for this to be `sa.Column("metadata", sa.JSON)`?

Or

`metadata = sa.Column("_metadata", sa.JSON)`

I'm not sure whether it's SQLite/PostgreSQL that disallow this or whether it's SQLAlchemy.

Reply:
needs to be `_metadata` as `metadata` was existing property already, so `_metadata = sa.Column("_metadata", sa.JSON)` ?

Reply:
How about `_metadata = sa.Column("metadata", sa.JSON)`?